src/main/java/org/terasology/gookeeper/component/MatingComponent.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 MovingBlocks
```

Comment:
More trivial: outdated license header year (the template has been updated but may need an IDE refresh)

Reply:
2018 all the things!